File: java/src/Solution1184.java
public class Solution1184 {
	public int distanceBetweenBusStops(int[] distance, int start, int destination) {
		if (start > destination) {
			int temp = destination;
			destination = start;
			start = temp;
		}
		int sum = 0, sum1 = 0;
		for (int i = 0; i < distance.length; i++) {
			if (i >= start && i < destination) {
				sum1 += distance[i];
			}
			sum += distance[i];
		}
		int sum2 = sum - sum1;
		return Math.min(sum1, sum2);
	}
}

File: GoLang/solution/solution1184.go
package solution

func distanceBetweenBusStops(distance []int, start int, destination int) int {
	if start > destination {
		start, destination = destination, start
	}
	sum, sum1 := 0, 0
	for i := 0; i < len(distance); i++ {
		if i >= start && i < destination {
			sum1 += distance[i]
		}
		sum += distance[i]
	}
	sum2 := sum - sum1
	if sum2 <= sum1 {
		return sum2
	}
	return sum1
}
